crates/rolldown_common/src/inner_bundler_options/types/minify_options.rs

```diff
@@ -84,8 +84,8 @@ pub struct MinifyOptionsObject {
 impl From<&MinifyOptionsObject> for oxc::minifier::MinifierOptions {
   fn from(value: &MinifyOptionsObject) -> Self {
     Self {
-      mangle: value.mangle.then(MangleOptions::default),
-      compress: Some(oxc::minifier::CompressOptions::default()),
+      mangle: value.mangle.then_some(MangleOptions { top_level: true, debug: false }),
+      compress: Some(oxc::minifier::CompressOptions::all_true()),
```

Comment:
Please be careful, this drops console.log and debuggers and sets the latest `target`. This is probably not what we want.